platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/roster/InvalidAddressBookException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.roster;
+
+/**
+ * An exception thrown by the RosterValidator when a given Roster is invalid.
```

Comment:
Typo: 
```suggestion
 * An exception thrown by the RosterValidator when a given AddressBook  is invalid.
```

Reply:
Nice catch! Thanks! I'll fix this typo in a follow-up PR in the next few days.

Reply:
I'm tracking it here: https://github.com/hashgraph/hedera-services/issues/14969#issuecomment-2316340042